spec/mongo/session_spec.rb

```diff
@@ -268,6 +268,18 @@
         end.to raise_error(Mongo::Error::SessionEnded)
       end
     end
+
+    context "when the sesion is not materialized" do
+      let(:session) { authorized_client.get_session(implicit: true) }
+
+      it "raises SessionNotMaterialized" do
+        expect(session.materialized?).to be false
```

Comment:
Can this line be in a `before` block to separate this precondition check from the actual test please?